### ISSUE TITLE
Element is trying to record audio without consent + usage of trackers

### ISSUE THREAD

**Issue by joe540** [vector-im/element-android#3035]
According to XPrivacyLua, Element is trying to record audio from my phone even if I am not actively using it (i.e. I am not asking it to record a voice clip or anything)

It also use trackers all the time

Isn't Element supposed to be privacy friendly and let you control your data???????

**Comment by ShadowJonathan** [comment]
Can you please explain which trackers Element is using on your device? Also, if it's possible, detail when or where Element tried to record audio on your device.

**Comment by bmarty** [comment]
Yes, can you provide more details about what XPrivacyLua is reporting please?

**Comment by joe540** [comment]
Hello,

    Unfortunately I don't have much info because XPrivacyLua just tell me when the app last used some privacy permissions. Here's what it shows:

![Screenshot_20210323-163128_XPrivacyLua](https://user-images.githubusercontent.com/22401071/112214672-d9aa5800-8bf5-11eb-8bab-71a9e4039ce6.png)

When I open the app, it uses the microphone somehow. Anytime the app is open, it also use trackers (but I don't know what XPrivacyLua concider as tracking... (I think it's when the app is trying to access ROM's Build properties, network/SIM, country/operator, etc)

I Installed XPrivacyLua not so long ago and I started blocking what I thought apps didn't need to protect my privacy and I was concerned because Element is supposed to be one of those privacy friendly app

If I can help by providing any additional information, please let me know

**Comment by ShadowJonathan** [comment]
It may be checking for microphone/video access on startup 🤔
